AK/String.cpp

```diff
@@ -278,8 +278,11 @@ bool String::starts_with(char ch) const
     return characters()[0] == ch;
 }
 
-bool String::ends_with(const StringView& str) const
+bool String::ends_with(const StringView& str, CaseSensitivity case_sensitivity) const
 {
+    if (case_sensitivity == CaseSensitivity::CaseInsensitive)
+        return StringUtils::ends_with_ignoring_case(*this, str);
```

Comment:
Let's move the whole `ends_with()` to `StringUtils` so it can be used from `FlyString` and `StringView` as well :)

Reply:
Sure, done.